bash.exe.stackdump

```diff
@@ -0,0 +1,19 @@
+Stack trace:
```

Comment:
This file should not be in the repo, you can add `bash.exe.stackdump` to `.gitignore` if it keeps on showing up